src/renderer/util/theme.js

```diff
@@ -1,10 +1,16 @@
-import { THEME_STYLE_ID, COMMON_STYLE_ID, DEFAULT_CODE_FONT_FAMILY, oneDarkThemes, railscastsThemes } from '../config'
+import { isLinux, THEME_STYLE_ID, COMMON_STYLE_ID, DEFAULT_CODE_FONT_FAMILY, oneDarkThemes, railscastsThemes } from '../config'
 import { dark, graphite, materialDark, oneDark, ulysses } from './themeColor'
 
 const patchTheme = css => {
   return `@media not print {\n${css}\n}`
 }
 
+const getEmojiFallbackFontFamily = () => {
```

Comment:
the return value is `style`, not font family, so pls rename the function to others.